### PR TITLE
runnodes.bat no longer recursives when creating nodes.

### PR DESCRIPTION
**Pull request by Clintonio**
Fixes issue #128. 

I just need one person with a Windows machine to review this one.

### CODE REVIEW

File: publish.properties [hunk 1]
```diff
@@ -1 +1 @@
-gradlePluginsVersion=0.6.5
```
Comment (mnesbit) [inline publish.properties]: Major version number change? Also, how is this aligned with corda-template?

File: gradle-plugins/cordformation/src/main/resources/net/corda/plugins/runnodes.bat [hunk 2]
```diff
@@ -3,7 +3,7 @@
 REM Change to the directory of this script (%~dp0)
 Pushd %~dp0
 
-FOR /R ".\" %%G in (.) DO (
```
Comment (mnesbit) [inline gradle-plugins/cordformation/src/main/resources/net/corda/plugins/runnodes.bat]: I know this PR is only for the bug fix, so this is fine. In future we do also need to add a line to set the fat jar unpack location and to support remote debug mode.

File: publish.properties (hunk 1 above)
Comment (Clintonio) [inline publish.properties]: Yeah, jumping up to match with the Corda version (felt it wasn't worth it at the time of M7).

File: gradle-plugins/cordformation/src/main/resources/net/corda/plugins/runnodes.bat (hunk 2 above)
Comment (Clintonio) [inline gradle-plugins/cordformation/src/main/resources/net/corda/plugins/runnodes.bat]: Agree. I'll make an issue for it and assign myself.

**Comment by Clintonio** [comment]
@mnesbit back in your court.

**Comment by mnesbit** [comment]
Looks simple enough to me.

**Pull request merged**